api/email/unsubscribe/[unsubscribeToken].ts

```diff
@@ -0,0 +1,61 @@
+import assert from 'assert';
+
+import { VercelRequest, VercelResponse } from '@vercel/node';
+
+import { decodeToken, isEmailType } from '../../../api-lib/email/unsubscribe';
+import { adminClient } from '../../../api-lib/gql/adminClient';
+import { NotFoundError, errorResponse } from '../../../api-lib/HttpError';
+
+export default async function handler(req: VercelRequest, res: VercelResponse) {
+  let unsubscribeToken: string | undefined;
+  if (typeof req.query.unsubscribeToken == 'string') {
+    unsubscribeToken = req.query.unsubscribeToken;
+  } else if (Array.isArray(req.query.unsubscribeToken)) {
```

Comment:
do we need this? should never be an array, right?

Reply:
@crabsinger I think so but typescript is expecting either string or string[] so I handled it the same way api/cosoul/[address] is handled